src/opera/commands/__init__.py

```diff
@@ -1,4 +1,5 @@
 from opera.commands import (
+    unpackage,
```

Comment:
Could you sort this alphabetically? It's the only order that makes sense, i think.

Reply:
Yes, maybe it would be better. I tried to sort the commands as they would be executed normally.

Reply:
Sorted.